input/docs/handbook/data-binding/xamarin-forms.md

```diff
@@ -67,7 +79,43 @@ public partial class TheContentPage : ReactiveContentPage<TheViewModel>
 }
 ```
 
-# Routing
+The ReactiveUI binding engine is very powerful, but there may be cases where you want to bind to C# Events coming from your view (such as when a buttons `Tap` event fires). For this scenario use [Pharmacist](https://github.com/reactiveui/Pharmacist), which automatically creates Observables for all the C# Events in your project.
```

Comment:
It's also worth noting 
- ReactiveUI bindings don't use the Xamarin.Forms XAML Binding Engine
- XAML is still an option, you don't need to use ReactiveUI bindings
- You can use both in the same application

Reply:
Added that in